.github/actions/generic-ci/action.yml

```diff
@@ -122,7 +122,7 @@ runs:
       if: runner.os == 'Windows'
       shell: powershell
       working-directory: ${{github.workspace}}
-      run: echo "PATH=$env:PATH;$(pwd)\dependencies\install\bin\;$(pwd)\install\bin\;"| Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+      run: echo "PATH=$env:PATH;$(pwd)\dependencies\install\bin\;$(pwd)\dependencies\install\lib\;$(pwd)\install\bin\;"| Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
```

Comment:
can we fix that in boost or usd install instead ?

Reply:
Why would we want to fix it? It's legit, "modules" (dynamically loaded libs) on Windows are installed in `lib`, not `bin`

Reply:
dlls are expected to be in bin, not lib.

Reply:
No. DLLs are in `bin` if the libraries are `SHARED`, but in `lib` if the libraries are `MODULE`.

Reply:
See https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#library-output-artifacts

Reply:
https://stackoverflow.com/questions/4845984/difference-between-modules-and-shared-libraries

Reply:
Any idea why we are getting a module ?

Why does it matter to put in the PATH then ?

Reply:
still interested by the answer here

Reply:
A module on Windows is a DLL loaded dynamically and has no `.lib` associated import file. It's intended to be loaded by the `LoadLibrary` API. We have to put it in the PATH because Windows is looking in it when calling `LoadLibrary`.

Reply:
ok got it.